examples/head_or_tails.py

```diff
@@ -0,0 +1,15 @@
+import random
+
+from nextcord.ext import commands
+
+bot = commands.Bot(command_prefix="$")
+
+@bot.command()
+async def headortails(ctx, answer=None):
+
```

Comment:
This space should be removed. Also possibly add some comments to the top of the file to explain what it does?

Reply:
Okay!

Reply:
Done.